deploy/platform/kubernetes/Chart.yaml

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+name: skywalking-showcase
+home: https://skywalking.apache.org
+version: 0.0.0
+description: Apache SkyWalking showcase
+icon: https://raw.githubusercontent.com/apache/skywalking-kubernetes/master/logo/sw-logo-for-chart.jpg
+sources:
+  - https://github.com/apache/skywalking-showcase
+dependencies:
```

Comment:
> How do we lock the version of helm repo? I can't find the relative changes of make.in in the root.


@wu-sheng most of the versions are still maintained in the `Makefile.in` file, only the Helm Chart versions in this files are different, they have to be changed in this file, no variable substitutions can be done in `Chart.yaml` file here

Reply:
From the content here, these versions are Helm Chart versions themselves, not the app version (like `OAP` version), so in fact the previous versions (before this PR) are still the same as before

Reply:
OK, as this is a new place. Let's add a doc about how to bump up component versions, especially for SkyWalking components. 

@wankai123 @kezhenxu94 Another thing about OTEL collector version, we are not bumping up for a while, there is a big gap, we should doc them and try to bump them.

Reply:
> @wankai123 @kezhenxu94 Another thing about OTEL collector version, we are not bumping up for a while, there is a big gap, we should doc them and try to bump them.

opentelemetry-collector:0.72.0
kube-state-metrics:v2.8.0

I'm using this version with no problems

Reply:
@kezhenxu94 Could we upgrade this as well?

Reply:
> @kezhenxu94 Could we upgrade this as well?

Sure

Reply:
Currently the kube-state-metrics is latest 5.0.1

Reply:
I can update the otel version